rules/false_positives/rancher.yara

```diff
@@ -0,0 +1,14 @@
+rule pull_script : override {
+  meta:
+    curl_chmod_relative_run_tiny = "high"
```

Comment:
since it's a known-good, let's say medium

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).